ocd_backend/transformers/greenvalley.py

```diff
@@ -53,9 +53,7 @@ def _get_documents_as_media_urls(self, original_item):
 
                 url = "https://staten.zuid-holland.nl/dsresource?objectid=%s" % att['objectid']
 
-                doc_hash = sha1(url.encode('utf-8') +
-                                ':'.encode('utf-8') +
-                                att['objectname'].encode('utf-8')).hexdigest()
+                doc_hash = sha1((url + ':' + att['objectname']).encode('utf-8')).hexdigest()
```

Comment:
I think we should prefer f-strings instead of concats now:

```suggestion
                doc_hash = sha1(f"{url}:{att['objectname'])}".encode('utf-8')).hexdigest()
```